docs/package.json

```
@@ -31,18 +31,18 @@
     "@fortawesome/fontawesome-svg-core": "^6.4.0",
     "@fortawesome/free-solid-svg-icons": "^6.4.0",
     "@fortawesome/react-fontawesome": "^0.2.0",
-    "@mui/base": "5.0.0-beta.4",
-    "@mui/docs": "^5.12.4",
+    "@mui/base": "^5.0.0-beta.5",
```

```suggestion
    "@mui/base": "5.0.0-beta.5",
```

We should remove `^` on all alpha & beta releases manually